src/components/media-views.js

```diff
@@ -700,13 +706,33 @@ AFRAME.registerComponent("media-video", {
       } else {
         videoEl.src = url;
         videoEl.onerror = reject;
+
+        if (this.data.audioSrc) {
+          // If there's an audio src, create an audio element to play it that we keep in sync
+          // with the video while this component is active.
+          audioEl = await createVideoOrAudioEl("audio");
+          audioEl.src = this.data.audioSrc;
+          audioEl.onerror = reject;
+
+          this._audioSyncInterval = setInterval(() => {
+            if (Math.abs(audioEl.currentTime - videoEl.currentTime) >= 0.33) {
+              // In Chrome, drift of a few frames seems persistent
+              audioEl.currentTime = videoEl.currentTime;
+            }
+
+            if (videoEl.paused !== audioEl.paused) {
+              videoEl.paused ? audioEl.pause() : audioEl.play();
+              audioEl.currentTime = videoEl.currentTime;
```

Comment:
Why do this again here? You would have just synced it above it they had diverged at all.

Reply:
I did it here because it gives us an opportunity to correct any drift that's accrued but still below the amount we require to do a re-sync.